src/backend/providers/playnite/PlayniteProvider.cpp

```diff
@@ -0,0 +1,307 @@
+// Pegasus Frontend
+// Copyright (C) 2017-2020  M�ty�s Mustoha
```

Comment:
Found another encoding error.

Reply:
Weird that find/replace didn't find that.  I wonder if it got re-encoded weird somehow.  Still don't know what's causing it